src/neural/network_check.cc

```diff
@@ -66,7 +85,10 @@ class CheckComputation : public NetworkComputation {
   }
 
  private:
-  void Check() {
+  static constexpr int kNumOutputPolicies = 1858;
+  const CheckParams& params_;
+
+  void CheckOnly() {
```

Comment:
const?

Reply:
Is that really important on private members? Yet addressed.